List_of_Users_Project/userList.py

```diff
@@ -0,0 +1,70 @@
+import requests
+from datetime import datetime, timedelta
+import time
+import json
+import calendar
+import csv
+import sys
+import os
+import argparse
+import urllib3
+
+#to disable warnings for not verifying host
+urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
+
+class List:
```

Comment:
This is not a good name for a class since it may cause confusion with built-in classes.  Change to something more descriptive like UserList or similar